setup.py

```diff
@@ -46,16 +47,17 @@ def remove_bdist_rpm_source_file():
 shared_setup.remove_extra_bdist_rpm_files = remove_bdist_rpm_source_file
 shared_setup.SHARED_TARGET.update({
     'url': 'https://github.com/hpcugent/vsc-base',
-    'download_url': 'https://github.com/hpcugent/vsc-base'
+    'download_url': 'https://github.com/hpcugent/vsc-base',
+    'zip_safe': True,
 })
 
 
 PACKAGE = {
     'name': 'vsc-base',
-    'version': '2.2.6',
+    'version': '2.3.0',
```

Comment:
no need to bump the version to 2.3 , nothing in the code changed. You can just bump the release, since that's the only thing that changed.